apis/builds.json

```diff
@@ -0,0 +1,610 @@
+              },
+              "type":"object"
+            }
+          },
+          "404":{
+            "description":"Not found",
+            "schema":{
+              "$ref":"#/definitions/Error"
+            }
+          },
+          "default":{
+            "description":"Unexpected error",
+            "schema":{
+              "$ref":"#/definitions/Error"
+            }
+          }
+        },
+        "summary":"Get List of Builds",
+        "tags":[
+          "Build"
+        ]
+      }
+    },
+    "/{build_source}/{build_id}/":{
+      "get":{
+        "operationId":"get_build_v2",
+        "parameters":[
+          {
+            "$ref":"#/parameters/buildSource"
+          },
+          {
+            "$ref":"#/parameters/buildId"
+          },
+          {
+            "$ref":"#/parameters/buildUserId"
+          },
+          {
+            "$ref":"#/parameters/buildOrgId"
+          },
+          {
+            "$ref":"#/parameters/buildGroupId"
+          },
+          {
+            "$ref":"#/parameters/buildTeamId"
+          }
+        ],
+        "responses":{
+          "200":{
+            "description":"Build",
+            "schema":{
+              "$ref":"#/definitions/BuildV2"
+            }
+          },
+          "404":{
+            "description":"Not found",
+            "schema":{
+              "$ref":"#/definitions/Error"
+            }
+          },
+          "default":{
+            "description":"Unexpected error",
+            "schema":{
+              "$ref":"#/definitions/Error"
+            }
+          }
+        },
+        "summary":"Get Build detail",
+        "tags":[
+          "Build"
+        ]
+      }
+    },
+    "/{build_source}/jobs/{job_id}/build/":{
+      "get":{
+        "operationId":"get_build_by_jobId_v2",
+        "parameters":[
+          {
+            "$ref":"#/parameters/buildSource"
+          },
+          {
+            "$ref":"#/parameters/buildJobId"
+          },
+          {
+            "$ref":"#/parameters/buildUserId"
+          },
+          {
+            "$ref":"#/parameters/buildOrgId"
+          },
+          {
+            "$ref":"#/parameters/buildGroupId"
+          },
+          {
+            "$ref":"#/parameters/buildTeamId"
+          }
+        ],
+        "responses":{
+          "200":{
+            "description":"Build",
+            "schema":{
+              "$ref":"#/definitions/BuildV2"
+            }
+          },
+          "404":{
+            "description":"Not found",
+            "schema":{
+              "$ref":"#/definitions/Error"
+            }
+          },
+          "default":{
+            "description":"Unexpected error",
+            "schema":{
+              "$ref":"#/definitions/Error"
+            }
+          }
+        },
+        "summary":"Get Build Jobs",
+        "tags":[
+          "Build"
+        ]
+      }
+    },
+    "/{build_source}/{build_id}/jobs/":{
+      "get":{
+        "operationId":"get_builds_jobs_v2",
```

Comment:
same for this as well, `get_builds_jobs_v2` operationId can be used as `getBuildsJobsV2`.

Reply:
Thanks for explanation 🙇